src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -188,6 +188,105 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
+        <fullName>InstallmentFrequency__c</fullName>
+        <description>Defines the installment schedule of this Recurring Donation along with Installment Period. For example, if Installment Period is Monthly and Installment Frequency is 3, the schedule would be defined as &quot;every 3 months&quot;.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Defines the installment schedule of this Recurring Donation along with Installment Period. For example, if Installment Period is Monthly and Installment Frequency is 3, the schedule would be defined as &quot;every 3 months&quot;.</inlineHelpText>
+        <label>Installment Frequency</label>
+        <precision>18</precision>
```

Comment:
@sam-knox What would you think about making this field a size of 3 so that cannot support a value beyond 999? As an Interval, it's meant to define something like "Every 3 Months" or "Every 90 Days". I can't see there ever being a case where we'd want a value over 100, but I'm hesitant to set a max size of 2.

Reply:
@force2b - Yeah, that's a good call. 3 sounds like a good number. Then users can indicate "every 180 days" or whatever if they need to. We probably don't need to support "every 78,560 days" though!

Reply:
Thanks @sam-knox 
@meena-sfdc Can you make this change as well?

Reply:
The code review changes are done and ready for review. The helptext for Enhanced recurring donations needs to be decreased in characters as it exceeds the limit.

Reply:
@meena-sfdc - Please use this text for the Enhanced Recurring Donations Help Text and Description. It's under the character limit now:

"This option enables Enhanced Recurring Donations. WARNING: When you enable this option, NPSP changes your existing Recurring Donations records and the Recurring Donations object to match the data model for Enhanced Recurring Donations."

Reply:
Thanks, @sam-knox . I have made the changes and it looks good now.